sklearn/ensemble/tests/test_forest.py

```diff
@@ -1330,3 +1330,61 @@ def test_forest_degenerate_feature_importances():
     gbr = RandomForestRegressor(n_estimators=10).fit(X, y)
     assert_array_equal(gbr.feature_importances_,
                        np.zeros(10, dtype=np.float64))
+
+
+@pytest.mark.parametrize('name', FOREST_CLASSIFIERS_REGRESSORS)
+@pytest.mark.parametrize(
+    'max_samples_exception',
```

Comment:
Could you add an additional parameter to check that the error matches the message?
It only requires a partial string and not the full message.

```suggestion
    'max_samples, err_type, err_msg',
```